src/components/inputs/buttons/MenuButton/MenuButton.tsx

```diff
@@ -5,19 +5,26 @@ import './styles.css';
 function MenuButton({
   show,
   setShow,
-  icon,
+  children,
 }: {
   show: boolean;
   setShow: Dispatch<SetStateAction<boolean>>;
-  icon: JSX.Element;
+  children?: JSX.Element;
 }): JSX.Element {
   return (
     <>
       <button
-        className={!show ? 'round-button' : 'round-button white-background'}
+        className={
+          !show
+            ? 'outline-button hover'
```

Comment:
the profile button differs from the outline button: figma shows 2px border width and 6 px radius

Reply:
updated